lib/faraday_middleware/response/follow_redirects.rb

```diff
@@ -104,10 +116,29 @@ def update_env(env, request_body, response)
     end
 
     def follow_redirect?(env, response)
+      return true if follow_meta_refresh?(env)
+
       ALLOWED_METHODS.include? env[:method] and
         REDIRECT_CODES.include? response.status
     end
 
+    def follow_meta_refresh?(env)
+      return false unless @options[:follow_meta_refresh]
+
+      meta_refresh_tag(env).size > 0
+    rescue Nokogiri::SyntaxError
```

Comment:
Actually this line doesn't invoke Nokogiri...

I was little unsure about that, then now I checked doing exactly that:
- Commented the spec I wrote that was using it
- Commented the require 'nokogiri'
- Commented nokogiri at Gemfile 

Then I run the spec suite and everything passed...

Since I have Ruby 2.5.1 on my machine, I was unsure about if that behaviour is the same for all Ruby versions that faraday_middleware aims to support, so, I left a branch on my repo with those comments and sent that to CI, so the results are here:
https://travis-ci.org/efqdalton/faraday_middleware

There I saw that 3 specs are broken... But correct me if i'm wrong, but all of those broke not because of Nokogiri, but because quality threshold was crossed.

```
Quality threshold crossed (1):
  coverage/covered_percent is 98.12, should be >= 98.5
```

Then I think that this is evaluated only if `@options[:follow_meta_refresh]` is truthy or other bizarre cases like `@options` was mistakenly given as not a Hash so it may throw if not respond to method `[]`.